src/plugins/cosmos/CosmosAsset.tsx

```diff
@@ -12,10 +12,14 @@ import { useAppDispatch, useAppSelector } from 'state/store'
 
 import { CosmosAssetAccountDetails } from './CosmosAssetAccountDetails'
 
+export interface MatchParams {
```

Comment:
```suggestion
export type MatchParams {
```

Reply:
nitpick - but we only use types. interface should only be used where they define something that is then implemented elsewhere